logfire/_internal/main.py

```diff
@@ -1451,12 +1452,33 @@ def instrument_sqlite3(
             },
         )
 
+    def instrument_aws_lambda(
+        self, lambda_function: LambdaFunction, **kwargs: Unpack[AwsLambdaInstrumentKwargs]
```

Comment:
```suggestion
        self, lambda_function_handler: LambdaFunctionHandler, **kwargs: Unpack[AwsLambdaInstrumentKwargs]
```

Reply:
The examples on AWS use the `lambda_function` name for the function, I prefer to use the same notation.

Reply:
> A function handler can be any name; however, the default name in the Lambda console is lambda_function.lambda_handler. This function handler name reflects the function name (lambda_handler) and the file where the handler code is stored (lambda_function.py).

https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html

Reply:
ah yeah, `lambda_handler`, not `lambda_function`. mb

Reply:
Anything else in this PR besides this? I'll merge after fixing this otherwise.